_posts/2019-12-09-PC-Engines-Firmware-v4-11-0-1.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+title:  "PC Engines apu coreboot Open Source Firmware v4.11.0.1"
+date:   2019-12-09
+categories: Firmware
+---
+# PC Engines apu coreboot Open Source Firmware v4.11.0.1
+
+## Key changes
+
+Mainline:
+
+1. **Rebased** with official coreboot repository **commit** 9f56eed.
+2. **Temperature is now being showed in pfSense** dashboard.
+
+
+## coreboot community
+
+Patches sent for review:
```

Comment:
@miczyg1 please ping me when you add patch links and merge the post

Reply:
Done